pyshtools/shclasses.py

```diff
@@ -2163,7 +2163,52 @@ def plot(self, show=True, fname=None):
         fname : str, optional, default = None
             If present, save the image to the file.
         """
-        fig, ax = self._plot()
+        if coordinates == 'grid':
+            fig, ax = self._plot()
+        elif coordinates == 'cylindrical':
+            # plot regular lat/lon image
+            from .utils.grid_interpolator import interpolate_2dgrid  # NOQA
```

Comment:
I don't know, my style checker told me to, because it was a late import I guess... This isn't required?